From 918b3831884de099c48f41554f9e043e45203a0c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 04 Jun 2019 18:54:06 +0300
Subject: [PATCH] [SW-0] pll: move clk gater offset to common space

The commit:
a8a4753 ("[SW-4185] gaudi: pll: add hbw clk sw glitch w/a")
added a private implementation of HL_PLL_GATER_OFFSET macro while in
fact the macro should have been added in the common space in pll.h
header.
Move the private macro to the common space in pll.h header and also
rename it to align with the others and according to the documentation
(SoC online).

Change-Id: I2a86987837578667ecb509a14af5dcf5c5cd93dd
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.h b/common/pll.h
index 2bd0927..0610057 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -23,6 +23,7 @@
 #define HL_PLL_DIV_FACTOR_CMD	0x220
 #define HL_PLL_DIV_SEL		0x280
 #define HL_PLL_DIV_EN		0x2a0
+#define HL_PLL_CLK_GATER	0x300
 
 #define HL_PLL_MAX_WAIT_TIME	20	/* 20 us */
 #define HL_PLL_LOCK_TOUT	5000	/* 5 ms */
diff --git a/gaudi/asic.c b/gaudi/asic.c
index f6bc30c..8932694 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -341,15 +341,13 @@
 		preboot_puts("Surprise! Woke up from WFE...\n");
 }
 
-#define HL_PLL_GATER_OFFSET	0x300
-
 static void hl_pll_hbw_gated_clk_sw(struct hl_pll_cfg *cfg)
 {
-	regmap_writel(0xE, cfg->base + HL_PLL_GATER_OFFSET);
+	regmap_writel(0xE, cfg->base + HL_PLL_CLK_GATER);
 	hl_pll_udelay(1);
 	regmap_writel(HL_PLL_PLL_CLK, cfg->base + HL_PLL_DIV_SEL);
 	hl_pll_udelay(10);
-	regmap_writel(0xF, cfg->base + HL_PLL_GATER_OFFSET);
+	regmap_writel(0xF, cfg->base + HL_PLL_CLK_GATER);
 	hl_pll_udelay(10);
 }
 
